RaveAddIn/XML/brat.xml

```diff
@@ -82,16 +82,16 @@
           </Node>
           <Node label="Vegetation">
             <Children>
-              <Node label="Existing Vegetation Type" xpath="Raster[@id='EXVEG']" type="raster" symbology="ExistVeg" />
-              <Node label="Historic Vegetation Type" xpath="Raster[@id='HISTVEG']" type="raster" symbology="HistVeg" />
+              <Node label="Existing Vegetation Type" xpath="Raster[@id='EXVEG']" type="raster" symbology="ExistVeg" transparency="40"/>
+              <Node label="Historic Vegetation Type" xpath="Raster[@id='HISTVEG']" type="raster" symbology="HistVeg" transparency="40"/>

```

Comment:
These transparency additions are good. Just noting that this business logic needs to be expanded to include all the stuff in the PyBRAT layer packages.  My understanding is @MattReimer is close to a fix on getting attribute tables in. Fix in another branch and pull request.

Reply:
Yup. We have them now. Will need to propagate but we have the technology

Reply:
Agreed. This looks good.